src/site.rs

```diff
@@ -43,7 +54,7 @@ impl Site {
         if file_basenames.len() == total {
             Ok(())
         } else {
-            Err("Some sites are duplicates of each other".to_string())
+            Err("Some sites / names are duplicates of each other".to_string())
```

Comment:
This error message is probably hard to understand.

Maybe something like this?

```suggestion
            Err("Some sites are duplicates of each other or result in the same filename".to_string())
```

Reply:
Ah, much better!